packages/web/src/components/chart/ReactiveChart.js

```diff
@@ -343,8 +360,7 @@ class ReactiveChart extends React.Component {
 
 ReactiveChart.generateQueryOptions = (props, after, value = {}) => {
 	const queryOptions = getQueryOptions(props);
-	const valueArray = Array.isArray(value) ? value : [value];
-	return getAggsQuery(valueArray, queryOptions, props);
+	return getAggsQuery(value, queryOptions, props);
```

Comment:
@SavvyShah Only make the above change when `chartType` is `custom`. No need to change the existing code.